### DISCUSSION TITLE
Kodi for stream client on firestick.

### DISCUSSION THREAD

**Discussion by blasco403** [GerryDazoo/Slinger#499] [General]
Hello, I was using VLC on firestick but was having issues with audio delay (fixed with a -250ms audio delay) but as the remote function was used, the delay for remote commands increased over time. A restart of the stream would fix the delay for a while.

I use Kodi for other streaming on firestick so I tried Kodi to stream from Slinger - it works great with no audio delay issues. I created .m3u files with the http string for my 2 slingbox's and then open those files with Kodi.

For those having issues with VLC, give it a try.

I side loaded Kodi (not from App Store) so I can control when it updates.

Both 19.5 and 20.0 versions of Kodi work great.

Good luck.

You have to turn on allow install from other sources on the Firestick, then use downloader application to download and install the arm7 32 bit version of Kodi from the official kodi.tv site.

**Comment by mike-uk** [comment]
@blasco403 Can you share the .m3u layout please? 

I tried it like this below and save as .m3u then load it on PVR IPTV Simple Client add-on on Kodi but it doesn't work. Which add-on do you use?
Thanks

---
#EXTM3U

#EXTINF:-1, Slingbox
http://192.168.1.123:8080/slingbox

---

**Reply by regmar505** [reply]
I tried what is described in this thread and it works very well #455

**Reply by mike-uk** [reply]
@regmar505 Thanks but not much luck with my set up. The picture just start and freeze while the sound keep playing. Back to vlc

**Reply by blasco403** [reply]
I don't use an addin, just native Kodi streaming. Just go to file and open the .m3u file you created. I add it to favorites so I can load the stream with a few clicks.

**Comment by blasco403** [comment]
The .m3u file just contains the URL you use to connect to your slinger server, nothing else. TTYLRonOn Apr 2, 2023, at 7:59 AM, mike-uk ***@***.***> wrote:﻿
@blasco403 Can you share the .m3u layout please? Thanks

**Reply by mike-uk** [reply]
@blasco403 Thanks It works once then stop. Back to vlc

**Reply by GerryDazoo** [reply]
Is there a reason why you don't want to use the Firestick version of SlingerPlayer?

**Reply by mike-uk** [reply]
@GerryDazoo I don't have a fire stick only android boxes with android 5.1.1 and chromecast 3

**Reply by GerryDazoo** [reply]
There's an Android version. Depending on the box, SlingerPlayer may work on that older android version.

**Reply by mike-uk** [reply]
@GerryDazoo I have tried on one of them but it wouldn't install. It does work on my phone but sceen is small. VLC is ok for now apart from when channel with the internal tuner takes a while. I am waiting for a set top box to arrived so should be able to change channel with out disturbing the stream then.

**Reply by GerryDazoo** [reply]
Rotate your phone 90 degrees to get a full screen picture

**Reply by blasco403** [reply]
Kodi connects quickly on my 350, but does a bunch of retries on my 500 and sometimes times out, have to select again and it usually connects then. Note sure why, some sort of negotiation problem I am guessing.

Once connected it works great thou.

I had the same issue of retries with VLC on my 500 as well.

**Reply by mike-uk** [reply]
@blasco403 Have you tried on VLC Disable hardware acceleration and turn on loop1 (on desktop) (on VLC for android loop lock + repeat 1) That's works well for me as long I don't change the channel too ofen as I have a built-in tuner on Pro-HD that stop and start the stream, for you it would be just buffer delay issue as you have component cable only in for 350 and 500.
